recipes/fftw/all/conandata.yml

```diff
@@ -9,6 +9,8 @@ patches:
   "3.3.9":
     - patch_file: "patches/fix-cmake.patch"
       base_path: "source_subfolder"
+    - patch_file: "patches/fix-simd.patch"
+      base_path: "source_subfolder"
   "3.3.8":
     - patch_file: "patches/fix-cmake.patch"
       base_path: "source_subfolder"
```

Comment:
Let's fix 3.3.8 as well, otherwise we will have options which are present in 3.3.8, but do not work in some cases.

Reply:
@mathbunnyru changes have been applied to 3.3.8. What do you mean with "May I ask you to add the PR in the header of the patch?", sorry?

Reply:
Explained below.